examples/monitoring/openai_proxy_quickstart_enterprise_mode.ipynb

```diff
@@ -0,0 +1,280 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Openai Proxy Quickstart - Enterprise Mode\n",
+    "\n",
+    "<a target=\"_blank\" href=\"https://colab.research.google.com/github/wandb/weave/blob/master/examples/monitoring/openai_proxy_quickstart_enterprise_mode.ipynb\">\n",
+    "  <img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/>\n",
+    "</a>\n",
+    "\n",
+    "A simple tool to capture metadata about your interactions with the OpenAI API.  Currently supports ChatCompletion endpoint, including streaming.\n",
+    "\n",
+    "Fundamentally, you can directly request data from the proxy using any technique you prefer (eg. `curl`), or use the openai client library. This notebook will highlight both apporaches"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# if not already installed\n",
+    "!pip install -qqq openai"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Settings & Setup\n",
+    "\n",
+    "The following setup the openai client to log correctly. Concretely, we need to:\n",
+    "1. Change your openai api base url.\n",
+    "2. Change your openai api key wandb api key.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Step 1: Configure target URL\n",
+    "WANDB_BASE_URL = \"https://api.wandb.ai\" # CHANGE ME!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Step 2: Ensure WANDB_API_KEY is set in env\n",
+    "import os\n",
+    "from getpass import getpass\n",
+    "\n",
+    "WANDB_APP_URL = WANDB_BASE_URL if WANDB_BASE_URL != \"https://api.wandb.ai\" else \"https://app.wandb.ai\"\n",
+    "WEAVE_ROOT_URL = WANDB_BASE_URL + \"/weave\" if WANDB_BASE_URL != \"https://api.wandb.ai\" else \"https://weave.wandb.ai\"\n",
+    "if os.getenv(\"WANDB_API_KEY\") is None:\n",
+    "    os.environ[\"WANDB_API_KEY\"] = getpass(f\"Paste your OpenAI key from: {WANDB_APP_URL}/authorize \\n\")\n",
```

Comment:
minor: this should say W&B API Key instead of OpenAI key

Reply:
thank you! fixed in next commit